syncmaster/schemas/v1/connections/s3.py

```diff
@@ -52,15 +56,22 @@ class S3UpdateConnectionSchema(S3BaseSchema):
     bucket_style: Literal["domain", "path"] | None = None
 
 
-class S3CreateAuthSchema(S3BaseSchema):
-    access_key: str
-    secret_key: SecretStr
+class CreateS3ConnectionSchema(CreateConnectionBaseSchema):
```

Comment:
I don't see there auth_data.type is set for S3 specifically

Reply:
Thought this would require generating a lot of different classes, but now I see it doesn't. Fixed